lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestFailedEvaluatorEventHandler.cs

```diff
@@ -65,8 +72,18 @@ public IConfiguration DriverConfigurations()
                 .Build();
         }
 
-        private sealed class FailedEvaluatorDriver : IObserver<IDriverStarted>, IObserver<IAllocatedEvaluator>, IObserver<ICompletedEvaluator>
+        private void ValidateSuccessForFailedEvaluator(string testFolder)
```

Comment:
Sounds good. I was copying off the model of another test. This is probably a better approach.